cheats/itemhandler.lua

```diff
@@ -8,67 +8,72 @@ local Logical = require(ItemHandler.rootPath .. "utility.logical")
 local Glossary = require(ItemHandler.rootPath .. "data.glossary")
 
 function ItemHandler.HasDefaultQuality(item)
-    for key, value in pairs(Glossary.ItemQuality) do
+    for key, value in pairs(Glossary.Quality) do
         if Logical.TableHasValue(Glossary.ForcedQuality[key], item) then
             return value
         end
     end
     return nil
 end
 
-function ItemHandler.SetItemLevel(itemdata)
-    local cs = System.CraftingSystem()
-    local result = cs:SetItemLevel(itemdata)
+function ItemHandler.SetLevel(data)
```

Comment:
I'd be in favour of keeping `itemdata` as the variable name here to distinguish from other objects (e.g. player data) and for consistency with naming in other functions.